calendar_v17/odoo_office365_calendar/models/calendar_event.py

```
@@ -0,0 +1,651 @@
+        return res
+
+
+
+    def unlink(self):
+        for rec in self:
+            if rec.o365_event_id and not self.env.context.get("no_outlook_unlink"):
+                office365 = ofs.Office365(rec.fetch_token())
+                try:
+                    office365.makerequest("DELETE", "events/" + rec.o365_event_id, {})
+                except BaseException:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)